src/ensembl/src/content/app/browser/browser-nav/BrowserNavBarRegionSwitcher.tsx

```diff
@@ -4,14 +4,18 @@ import { connect } from 'react-redux';
 import BrowserRegionEditor from '../browser-region-editor/BrowserRegionEditor';
 import BrowserRegionField from '../browser-region-field/BrowserRegionField';
 
+import { BreakpointWidth } from 'src/global/globalConfig';
+import { RootState } from 'src/store';
 import {
   toggleRegionEditorActive,
   toggleRegionFieldActive
 } from '../browserActions';
+import { getBreakpointWidth } from 'src/global/globalSelectors';
 
 import styles from './BrowserNavBarRegionSwitcher.scss';
 
 type Props = {
+  breakpointWidth: BreakpointWidth;
```

Comment:
From the point of view of this component, it probably just wants to know whether it should show both navigation components, or only one. Perhaps `display: full | compact` instead?

Reply:
>Perhaps display: full | compact instead?
How about `showBrowserRegionEditor`?

Reply:
If the only thing that this panel can do is show/hide the browser region editor field, I would say something like display: full/compact is simpler 🤷‍♂

Reply:
For this to be cleaner, won't we need an enum instead of using `full` and `compact` as strings? If that is the case it wouldn't be simple as using `showBrowserRegionEditor`.

Reply:
> For this to be cleaner, won't we need an enum instead of using full and compact as strings?

Yes, I was thinking of an enum :-)

Reply:
✅